pkgs/shells/nushell/plugins/bash-env.nix

```diff
@@ -0,0 +1,75 @@
+{
+  stdenv,
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  unstableGitUpdater,
+  makeWrapper,
+  resholve,
+  nushell,
+  bash,
+  jq,
+  coreutils,
+  IOKit,
+  Foundation,
+}:
+rustPlatform.buildRustPackage {
+  pname = "nu-plugin-bash-env";
+  version = "0.13.0";
+
+  src = fetchFromGitHub {
+    owner = "tesujimath";
+    repo = "nu_plugin_bash_env";
+    rev = "0.13.0";
+    hash = "sha256-fWFBGeNqrJj+fT8EusS8VhxoCmncqhQUplPEHOJlqRA=";
+  };
+
+  cargoHash = "sha256-z5pl1QjNLldXuj2ZWPlMEr/IeR9Fx73ZYDisIhEFxnY=";
+  passthru.updateScript = unstableGitUpdater { };
+
+  buildInputs = lib.optionals stdenv.isDarwin [
+    IOKit
+    Foundation
+  ];
```

Comment:
I'm not sure if this is needed as I don't own a Mac, just copied this from the other plugins.